pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -145,16 +132,22 @@ def set_world_ranks(self, process_idx: int = 0) -> None:
     def get_mp_spawn_kwargs(self, trainer: Optional["pl.Trainer"] = None) -> Dict[str, Any]:
```

Comment:
Do we still need this function?

Reply:
The original motivation for it to be a standalone method is that users/authors of plugins can just override only this part if they need to. If we remove it, they would instead override the spawn() method directly, or perhaps we could expose the `start_method` as an attribute to the plugin. Let's discuss it in an issue.

Reply:
#10895